### PR TITLE
increase parallel puller count

### PR DESCRIPTION
**Pull request by pongad**
The new default number of parallel pullers is NumCPU.
To keep up with publishers, we need to increase the number of pullers.
This commit increaes the number to 5*NumCPU,
the same as the old default.

### CODE REVIEW

File: load-test-framework/src/main/java/com/google/pubsub/clients/gcloud/CPSSubscriberTask.java [hunk 1]
```diff
@@ -39,10 +39,13 @@
...
     super(request, "gcloud", MetricsHandler.MetricName.END_TO_END_LATENCY);
-    this.subscription = SubscriptionName.create(
-        request.getProject(), request.getPubsubOptions().getSubscription());
+    this.subscription =
+        SubscriptionName.create(request.getProject(), request.getPubsubOptions().getSubscription());
     try {
-      this.subscriber = Subscriber.defaultBuilder(this.subscription, this).build();
+      this.subscriber =
+          Subscriber.defaultBuilder(this.subscription, this)
+              .setParallelPullCount(Runtime.getRuntime().availableProcessors() * 5)
```
Comment (mdietz94) [inline load-test-framework/src/main/java/com/google/pubsub/clients/gcloud/CPSSubscriberTask.java]: Why is this called parallel pull count when it is doing StreamingPull?

Comment (pongad) [reply load-test-framework/src/main/java/com/google/pubsub/clients/gcloud/CPSSubscriberTask.java]: I'm not quite sure what you're asking. We are using StreamingPull and the number determines how many connections we create to pull in parallel.

**Comment by pongad** [comment]
If this looks good to you, could you merge? I don't have write access :)

**Comment by pongad** [comment]
ping

**Pull request merged**